### PR TITLE
New header

### PR DESCRIPTION
**Pull request by ankita10r**
## Description

* Changes made to include X-CorrealtionId in the header for following API:
-  GSMA P2P
- International Remmitance Payee
- Collection Request
- Bulk transfer older
- Bulk Transfer
- Batch Transactions
- Bulk Account Lookup
- Batch Summary
- Batch Details
- Batch Lookup Details
- Get All Batches
- Notifications Callback
- Send Message to Message Gateway
- Message gateway provider callback

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Design related bullet points or design document link related to this PR added in the description above. 

- [x] Updated corresponding Postman Collection or Api documentation for the changes in this PR.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Add required Swagger annotation and update API documentation with details of any API changes if applicable

- [x] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

@avikganguly01 and @fynmanoj : please review

### CODE REVIEW

File: PostmanCollections/Collection OAF.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+					"mode": "raw",
+					"raw": "{\n    \"payer\": [\n        {\n            \"key\": \"MSISDN\",\n            \"value\": \"254708374149\"\n        },\n        {\n            \"key\": \"ACCOUNTID\",\n            \"value\": \"24450523\"\n        }\n    ],\n    \"amount\": {\n        \"amount\": \"1\",\n        \"currency\": \"USD\"\n    },\n    \"transactionType\": {\n        \"scenario\": \"MPESA\",\n        \"subScenario\": \"BUYGOODS\",\n        \"initiator\": \"PAYEE\",\n        \"initiatorType\": \"BUSINESS\"\n    }\n}",
+					"options": {
+						"raw": {
+							"language": "json"
+						}
+					}
+				},
+				"url": {
+					"raw": "{{ChannelHostName}}/channel/collection",
```
Comment (fynmanoj) [inline PostmanCollections/Collection OAF.postman_collection.json:50]: this API is not mentioned in description

File: PostmanCollections/Payment Hub.json [hunk 2]
```diff
@@ -43307,7 +43405,7 @@
 							}
 						],
 						"url": {
-							"raw": "{{OperationsHostName}}/api/v1/batch?batchId={{batchId}}",
+							"raw": "{{OperationsHostName}}/api/v1/batch?batchId={{batchId}}&X-CorrelationID={{clientCorrelationId}}",
```
Comment (fynmanoj) [inline PostmanCollections/Payment Hub.json:43408]: why are you passing correlation id in the URL?

Comment (ankita10r) [reply PostmanCollections/Payment Hub.json:43408]: I just added in the header, this gets added to the url automatically

File: PostmanCollections/Collection OAF.postman_collection.json (hunk 1 above)
Comment (ankita10r) [reply PostmanCollections/Collection OAF.postman_collection.json:50]: its named collection request (point 3 in description)

File: PostmanCollections/Payment Hub.json (hunk 2 above)
Comment (fynmanoj) [reply PostmanCollections/Payment Hub.json:43408]: that is strange

**Review by fynmanoj — APPROVED** [review]

**Pull request merged**